src/app/items/item-by-id.component.html

```diff
@@ -1,76 +1,92 @@
+      <ng-container *ngrxLet="fullFrameContentDisplayed$ as fullFrameContentDisplayed">
+        <alg-item-header [itemData]="itemData" *ngIf="!(fullFrameContent$ | async)"></alg-item-header>
+
+        <alg-access-code-view
+          *ngIf="showAccessCodeField$ | async"
+          i18n-sectionLabel sectionLabel="Access to activity"
+          i18n-buttonLabel="Button label for joining an item by (group) code" buttonLabel="Access"
+          [itemData]="itemData"
+          (groupJoined)="reloadItem()"
+        ></alg-access-code-view>
+
+        <alg-item-permissions
+          *ngIf="watchedGroup$ | async as watchedGroup"
+          [itemData]="itemData"
+          [watchedGroup]="watchedGroup"
+          (changed)="reloadItem()"
+        ></alg-item-permissions>
+
+        <alg-tab-bar *ngIf="!!(shouldDisplayTabBar$ | async) && !fullFrameContentDisplayed" />
+        <div
+          class="content-container"
+          [ngClass]="{
+            'no-top-padding': itemData.item.type === 'Task' && !!currentTab?.isTaskTab,
+            'side-padding': itemData.item.type === 'Task' && !currentTab?.isTaskTab && fullFrameContentDisplayed,
+            'right-padding': itemData.item.type === 'Task' && !currentTab?.isTaskTab && !fullFrameContentDisplayed
+          }"
+          #contentContainer
+          *ngrxLet="currentTab$ as currentTab"
+        >
+          <ng-container *ngIf="currentTab?.isTaskTab || (itemContentComponent?.isTaskLoaded$ | async) || currentTab?.tag === 'alg-content' || currentTab?.tag === 'alg-children-edit'">
+            <alg-error *ngIf="answerLoadingError$ | async as error; else noAnswerLoadingError">
+              <ng-container *ngIf="!error.fallbackLink; else fallback" i18n>Unable to load the answer</ng-container>
+              <ng-template #fallback>
+                <ng-container i18n>
+                  Unable to load the answer, <a [routerLink]="error.fallbackLink" class="alg-link">back to the regular task page</a>.
+                </ng-container>
+              </ng-template>
+              <p *ngIf="!error.isForbidden">{{ errorMessageContactUs }}</p>
+            </alg-error>
+
+            <ng-template #noAnswerLoadingError>
+              <ng-container *ngrxLet="taskConfig$ as taskConfig">
+                <div
+                  *ngIf="taskConfig && taskConfig.readOnly && currentTab && ['alg-content', 'task'].includes(currentTab.tag)"
+                  class="indicator"
+                  [ngClass]="{ 'no-left-padding': !fullFrameContentDisplayed }"
+                >
+                  <ng-container *ngIf="userProfile$ | async as userProfile">
+                    <alg-answer-author-indicator
+                      *ngIf="!!currentTab?.isTaskTab && taskConfig.initialAnswer"
+                      [answer]="taskConfig.initialAnswer"
+                      [userProfile]="userProfile"
+                      [itemData]="itemData"
+                      [isWatching]="isWatching"
```

Comment:
same with `isWatching` does it make sense to send it while the non-dump component could register to it?

Reply:
Fixed